packages/management/framework/bundle/config/pacakgeinstalls/addons-manager-service-account.yaml

```diff
@@ -0,0 +1,37 @@
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tanzu-addons-manager-package-sa
+  namespace: #@ data.values.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-group: "addons-manager-packageinstall/serviceaccount-0"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tanzu-addons-manager-package-cluster-role
+  annotations:
+    kapp.k14s.io/change-group: "addons-manager-packageinstall/serviceaccount-0"
+rules:
```

Comment:
i think you need to bubble up these rules to `tanzu-framework-package-role`, else when you try to install tkg package, it fails to install framework package as it doesn't have enough privileges

Reply:
Updated the clusterRole of framework packageInstall, changed to admin role.